src/css/controls/imports/videoThumbnail.less

```diff
@@ -30,6 +39,7 @@
         .jw-related-item-next-up & {
             height: 100%;
             width: auto;
+            background: #000000;
```

Comment:
what style bug does this help?

Reply:
If the video is vertical, we don't want to see the static thumb behind it (looks jarring)